gatherers/gatherersabc/gathererabc.py

```diff
@@ -0,0 +1,14 @@
+from abc import ABCMeta, abstractmethod
+from typing import List
+
+
+class Gatherer(metaclass=ABCMeta):
+
+    def __init__(self, suffixes: List[str], options: dict, extra: dict={}):
+        self.suffixes = suffixes
+        self.options = options
+        self.extra = extra
+
+    @abstractmethod
+    def gather(self):
+        pass
```

Comment:
Is this a stray file? Looks unused, and like an older copy of `gatherers/gatherersabc.py`.